book/setup/hello-world.md

```diff
@@ -0,0 +1,53 @@
+# Hello World
+
+To initiate a new React Native project you need to run: `react-native init <ProjectName>`. For this workshop, we have already done that for you and setup a skeleton project. To get started:
+
+```sh
+$ git clone https://github.com/winkerVSbecks/react-native-workshop.git
+$ cd react-native-workshop
+$ npm install
+```
+
+`react-native init` generates the following:
+- `index.ios.js`
+- `index.android.js`
+- iOS (Xcode) project
+- Android projects
+
+Let's open the project in a text editor to go through the generated code.
+
+
+## Bootstrapping
+
+In order to bootstrap a React Native app we use `AppRegistry` instead of `ReactDOM`, for example:
+
+```js
+import React, { AppRegistry, Component, View, Text } from 'react-native';
+
+class Root extends Component {
+  ...
+}
+
+AppRegistry.registerComponent('ApplicationName', () => Root);
+```
+
+
+## Run the App
+
+- iOS
+
+  ```sh
+  $ react-native run-ios
+  ```
+  or Open ``/Users/<userName>/reactNativeWorkshop/ios/reactNativeWorkshop.xcodeproj` in Xcode
+
+
+- Android
+
+  ```sh
+  $ react-native run-android
```

Comment:
It's worth pointing out you can't just do this if you have a default installation of Android Studio available. You need to do a custom install with Intel Atom system images and HAXM enabled. Might be worth just linking out here: https://facebook.github.io/react-native/docs/getting-started.html